src/data_collector/polygon_fundamentals_v2/validator_v2.py

```
@@ -58,7 +59,8 @@ def validate(self, resp: FundamentalDataResponse) -> ExtendedValidation:
                 denom = abs(assets) if assets else 1.0
                 if abs(assets - (liabilities + equity)) > 0.02 * denom:
                     cross_warnings.append("Balance sheet equation off by >2%")
-        except Exception:
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)